src/platform/user/authentication/utilities.js

```diff
@@ -232,6 +233,7 @@ export function sessionTypeUrl({
       : {};
 
   if (useOAuth && (isLogin || isSignup)) {
+    const validScope = scope || undefined;
```

Comment:
I don't think we need the `|| undefined` part of this var definition, won't it be undefined anyways if `scope` isn't defined?

Reply:
Makes sense, updated.

Reply:
thx looks like the update didn't push at that time updated.